regression/cbmc-java/generics_array_param1/GenericType.java

```diff
@@ -0,0 +1,10 @@
+class GenericType<T> {
```

Comment:
As mentioned these should be converted into unit tests added to `generate_concrete_generic_type/generate_java_generic_type.cpp`. As these symbols will no longer be generated except for when an entry function requiring constructing them is.